tools/bridge/bridge/main.py

```diff
@@ -234,6 +235,40 @@ def make_validator_balance_watcher(config, control_queue):
     )
 
 
+public_config_keys = (
+    "foreign_rpc_url",
+    "home_rpc_url",
+    "foreign_chain_max_reorg_depth",
+    "home_chain_max_reorg_depth",
+    "foreign_chain_token_contract_address",
+    "foreign_bridge_contract_address",
+    "home_bridge_contract_address",
+    "foreign_chain_event_fetch_start_block_number",
+    "home_chain_event_fetch_start_block_number",
+)
+
+
+def make_webservice(*, config, recorder):
+    d = config["webservice"]
+    if d["enabled"]:
+        ws = Webservice(host=d["host"], port=d["port"])
+    else:
+        ws = DummyWebservice()
+
+    def encode_address(v):
+        if isinstance(v, bytes):
+            return to_checksum_address(v)
+        else:
+            return v
+
+    public_config = {k: encode_address(config[k]) for k in public_config_keys}
```

Comment:
Not really happy with the `encode_address` function. What do you think about changing the way we deal with the config file to keep them in human readable form (i.e. not decode addresses in the `validate` functions, but rather in `main.py` where we use them)? Or add a `decode_config` function that transforms a human readable into a machine readable one?

Reply:
Yes, this was a quick hack and I'm also not happy with it.

I kind of like the decode_config idea, though I haven't thought it through. 
At this point I wonder if we should use marshmallow for the validation of the configuration.